f5_openstack_agent/lbaasv2/drivers/bigip/utils.py

```diff
@@ -53,16 +53,27 @@ def real_serialized(method):
         def wrapper(*args, **kwargs):
             """Necessary wrapper."""
             # args[0] must be an instance of iControlDriver
-            service_queue = args[0].service_queue
+            service_queue_map = args[0].service_queue_map
+            service_queue = service_queue_map["default"]
             my_request_id = uuid.uuid4()
 
             service = None
             if len(args) > 0:
                 last_arg = args[-1]
                 if isinstance(last_arg, dict) and ('loadbalancer' in last_arg):
                     service = last_arg
+
             if 'service' in kwargs:
                 service = kwargs['service']
```

Comment:
'service' is never in kwargs, and this service variable seems not important at all.

![image](https://user-images.githubusercontent.com/13101678/224960236-36bbc004-c851-450b-8aef-fe9412e48a58.png)

we need to find device info in service=args[-1].

But when here use it, args changes again.

![image](https://user-images.githubusercontent.com/13101678/224962026-2c307766-0a08-43a2-8a3e-d2a33f73b66d.png)

finally, we may put service as a keyword argument.